### PR TITLE
Header eb

### PR DESCRIPTION
**Pull request by ericberglund117**
### What does this PR do?
- Added styling to the Header component
- Added a font family to Index.html to use in welcome message
- Added a spy icon png to the header
- Added a background image of a desk to the header
- Created a background section to the header in order to separate elements 
- Updated header test after styling

### Where should the reviewer start?
- Look at Header component
- Review CSS to add/modify/remove anything 

### How should this be manually tested?
- npm start 
- header.test.js

### Any background context you want to provide?
- I would like to do more with the background and some more styling, but this is a base we can work with

### CODE REVIEW

File: src/components/Header/Header.test.js [hunk 1]
```diff
@@ -3,11 +3,27 @@ import Header from './Header.js'
...
+  it('should render a spy logo', () => {
+    render(<Header />);
+
+    const spyLogo = screen.getByAltText('spy-icon');
 
-    expect(welcomeMessage).toBeInTheDocument()
-  })
+    expect(spyLogo).toBeInTheDocument();
+  });
 })
```
Comment (timkeresey) [inline src/components/Header/Header.test.js:29]: Good call keeping tests up to date as we go.

**Review by timkeresey — COMMENTED** [review]
Good call keeping tests up to date as we go.

File: src/components/Header/Header.test.js [hunk 2]
```diff
@@ -3,11 +3,27 @@ import Header from './Header.js'
 import { render, screen } from '@testing-library/react'
```
Comment (taylorjohnson141) [inline src/components/Header/Header.test.js:3]: Thank you for doing the tests on these!

File: src/components/Header/Header.js [hunk 3]
```diff
@@ -1,9 +1,13 @@
 import './Header.css';
+import SpyIcon from '../../Images/spyIcon.png';
 
 const Header = () => {
   return (
-    <section>
+    <section className='header-section'>
+      <section className='background' title='background'>
+        <img alt='spy-icon' src={SpyIcon} />
```
Comment (taylorjohnson141) [inline src/components/Header/Header.js:8]: This looks really good!

File: src/components/Header/Header.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  display: flex;
+  justify-content: flex-start;
+  align-content: center;
+  width: 100%;
+  background: url(https://image.freepik.com/free-photo/overhead-shot-comfortable-workplace-with-notebook-white-desk-background_67155-3392.jpg) no-repeat;
+  background-size: cover;
+  margin-bottom: 1rem;
+}
+
+img[alt~='spy-icon'] {
```
Comment (taylorjohnson141) [inline src/components/Header/Header.css:12]: This is super cool and helpful!

**Pull request merged**